packages/c-flex/src/flex.ts

```diff
@@ -0,0 +1,59 @@
+import { h, defineComponent, PropType, computed } from 'vue'
+import { chakra, DOMElements, ThemingProps } from '@chakra-ui/vue-system'
+
+export type FlexProps = 'align' | 'justify' | 'wrap' | 'direction' | 'size'
+
+const CFlex = defineComponent({
+  props: {
+    as: {
+      type: [Object, String] as PropType<DOMElements>,
+      default: 'div',
+    },
+    row: {
+      type: Boolean,
+      default: false,
+    },
+    column: {
+      type: Boolean,
+      default: false,
+    },
+    reverse: {
+      type: Boolean,
+      default: false,
+    },
```

Comment:
In the more recent version of `@chakra-ui/styled-system` we're going to handle these prop values internally. So I think we should remove the `row`, `column`, `size`  and `reverse` props.

Along with the `as` prop, these are the props the `CFlex` component will accept:
https://github.com/chakra-ui/chakra-ui/blob/035250b78587761ba701ce4d66d6045a5d275aa1/packages/layout/src/flex.tsx#L66-L72

Reply:
As a future note, feel free to refer the [React repo](https://github.com/chakra-ui/chakra-ui) to guide you on some of the Chakra component patterns. 😉